firestore/tests/unit/test_cross_language.py

```diff
@@ -180,19 +181,18 @@ def test_create_testprotos(test_proto):
 @pytest.mark.parametrize('test_proto', _GET_TESTPROTOS)
 def test_get_testprotos(test_proto):
     testcase = test_proto.get
-    # XXX this stub currently does nothing because no get testcases have
-    # is_error; taking this bit out causes the existing tests to fail
-    # due to a lack of batch getting
-    try:
-        testcase.is_error
-    except AttributeError:
-        return
-    else:  # pragma: NO COVER
-        testcase = test_proto.get
-        firestore_api = _mock_firestore_api()
-        client, document = _make_client_document(firestore_api, testcase)
-        call = functools.partial(document.get, None, None)
-        _run_testcase(testcase, call, firestore_api, client)
+    firestore_api = mock.Mock(spec=['get_document'])
+    response = document_pb2.Document()
+    firestore_api.get_document.return_value = response
+    client, document = _make_client_document(firestore_api, testcase)
+
+    document.get()  # No '.textprotos' for errors, field_paths.
```

Comment:
As of today, the `.textproto` testcases in `google-cloud-common` contain only a [single file testing `Document.get`](https://github.com/googleapis/google-cloud-common/blob/03e4a2db3d229d4b97397e632ad831dd6a3a441f/testing/firestore/testdata/get-basic.textproto).  It does not pass any `field_paths`, nor is it marked as raising an error.